Sources/Build/PkgConfig.swift

```
@@ -0,0 +1,161 @@
+        try parser.parse()
+        if let cFlags = parser.cFlags {
+            // FIXME: handle spaces in paths.
+            self.cFlags += cFlags.characters.split(separator: " ").map(String.init)
+        }
+        if let libs = parser.libs {
+            // FIXME: handle spaces in paths.
+            self.libs += libs.characters.split(separator: " ").map(String.init)
+        }
+        
+        if(parser.dependencies.isEmpty) {
+            return
+        }
+        
+        for dep in parser.dependencies {
+            var pkg = try PkgConfig(name: dep)
+            try pkg.load()
+            self.cFlags += pkg.cFlags
+            self.libs += pkg.libs
+        }
+    }
+}
+
+private struct PkgConfigParser {
+    let pcFile: String
+    var variables = [String: String]()
+    var dependencies = [String]()
+    var cFlags: String?
+    var libs: String?
+    
+    enum Token {
+        case CFlats(String)
+        case Libs(String)
+        case Dependencies(String)
+        case Variable(name: String, value: String)
+    }
+    
+    init(pcFile: String) {
+        self.pcFile = pcFile
+    }
+    
+    mutating func parse() throws {
+        let file = File(path: self.pcFile)
+        for line in try file.enumerate() {
+            if !line.characters.contains(":") && line.characters.contains("=")  {
+                let equalsIndex = line.characters.index(of: "=")!
+                let name = line[line.startIndex..<equalsIndex]
+                let value = line[equalsIndex.successor()..<line.endIndex]
+                variables[name] = resolveVariables(value)
+            } else if line.hasPrefix("Requires: ") {
+                dependencies = parseDependencies(value(line: line))
+            } else if line.hasPrefix("Libs: ") {
+                libs = resolveVariables(value(line: line)).chomp()
+            } else if line.hasPrefix("Cflags: ") {
+                cFlags = resolveVariables( value(line: line)).chomp()
+            }
```

I'm not very familiar with the format, but I would recommend trying to get an `else` clause in here that will error out if we see something unexpected. This prevents the code from silently failing to work properly.